src/SmartComponents/Inventory/applications/compliance/ComplianceRemediationButton.js

```diff
@@ -55,29 +51,39 @@ class ComplianceRemediationButton extends React.Component {
     }
 
     render() {
+        const { disableRemediations } = this.props;
         return (
-            <React.Fragment>
-                <RemediationButton dataProvider={ this.dataProvider } />
-            </React.Fragment>
+            <div id='remediation-button' style={ { marginRight: '20px' } }>
```

Comment:
Instead of inlining any `px` values, can we target them with a `.scss` file and then use [this mixin](https://github.com/RedHatInsights/insights-frontend-components/blob/c254300608d93999d63ddb072cf812116ad5654a/src/Utilities/_mixins.scss#L3) to get a correct value?